airflow/www/templates/airflow/dags.html

```diff
@@ -238,320 +241,319 @@ <h2>DAGs</h2>
+    const ENTER_KEY_CODE = 13;
+
+    $('#tags_filter').select2({
+      placeholder: "Filter dags",
+      allowClear: true
+    });
+
+    $('#dag_query').on('keypress', function (e) {
+      // check for key press on ENTER (key code 13) to trigger the search
+      if (e.which === ENTER_KEY_CODE) {
+        var query = new URLSearchParams(window.location.search);
+        query.set("search", e.target.value.trim());
+        query.delete("page");
+        window.location = DAGS_INDEX + "?" + query.toString();
+        e.preventDefault();
+      }
+    });
 
-      $('#page_size').on('change', function() {
-        p_size = $(this).val();
-        window.location = DAGS_INDEX + "?page_size=" + p_size;
-      });
+    $('#page_size').on('change', function() {
+      p_size = $(this).val();
+      window.location = DAGS_INDEX + "?page_size=" + p_size;
+    });
 
-      function confirmTriggerDag(link, dag_id){
-        if (confirm("Are you sure you want to run '"+dag_id+"' now?")) {
-          postAsForm(link.href, {});
-        }
-        // Never follow the link
-        return false;
+    function confirmTriggerDag(link, dag_id){
+      if (confirm("Are you sure you want to run '"+dag_id+"' now?")) {
+        postAsForm(link.href, {});
       }
-
-      function confirmDeleteDag(link, dag_id){
-        if (confirm("Are you sure you want to delete '"+dag_id+"' now?\n\
-          This option will delete ALL metadata, DAG runs, etc.\n\
-          EXCEPT Log.\n\
-          This cannot be undone.")) {
-          postAsForm(link.href, {});
-        }
-        return false;
+      // Never follow the link
+      return false;
+    }
+
+    function confirmDeleteDag(link, dag_id){
+      if (confirm("Are you sure you want to delete '"+dag_id+"' now?\n\
+        This option will delete ALL metadata, DAG runs, etc.\n\
+        EXCEPT Log.\n\
+        This cannot be undone.")) {
+        postAsForm(link.href, {});
       }
-
-      var encoded_dag_ids = new URLSearchParams();
-
-      $.each($("[id^=toggle]"), function(i, v) {
-        var $input = $(v);
-        var dag_id = $input.attr('dag_id');
-        encoded_dag_ids.append('dag_ids', dag_id);
-
-        $input.change(function() {
-          var $buttons = $input.parent('.toggle').find('.btn');
-          $buttons.removeClass('btn-danger');
-          if ($input.prop('checked')) {
-            is_paused = 'true'
+      return false;
+    }
+
+    var encoded_dag_ids = new URLSearchParams();
+
+    $.each($("[id^=toggle]"), function(i, v) {
+      var $input = $(v);
+      var dag_id = $input.attr('dag_id');
+      encoded_dag_ids.append('dag_ids', dag_id);
+
+      $input.change(function() {
+        var $buttons = $input.parent('.toggle').find('.btn');
+        $buttons.removeClass('btn-danger');
+        if ($input.prop('checked')) {
+          is_paused = 'true'
+        } else {
+          is_paused = 'false'
+        }
+        var url = 'paused?is_paused=' + is_paused + '&dag_id=' + dag_id;
```

Comment:
```suggestion
        var url = 'paused?is_paused=' + is_paused + '&dag_id=' + encodeURIComponent(dag_id);
```

You probably missed this one https://github.com/apache/airflow/pull/8106#discussion_r404729864

Reply:
yeah, I pushed prematurely 😬

Reply:
Updated.